eng/pipelines/templates/send-to-helix.yml

```diff
@@ -0,0 +1,94 @@
+# Please remember to update the documentation if you make changes to these parameters!
```

Comment:
Loved the docs :).

Did we get these from some other repo? Mainly asking to see if we need to try to keep in sync.

Reply:
The template is from `eng/common/templates/steps/send-to-helix.yml`, slightly modified here.

Reply:
mmm I see, is there a way to instead use that template file and pass in some parameters to accommodate for our small modifications? If possible, I'd love to reduce the amount of yaml that we have to maintain.

Reply:
This `eng/common/templates/steps/send-to-helix.yml` is a sample template from arcade, and it's not used directly. But I can make changes in that file itself.

Reply:
Not sure what you mean when you say a "sample template". Tipically the templates under eng\common\templates are there so that repos can use them if they want/need as part of their build process. I don't want to block this PR on this, but I do want to make sure this is fixed. Can we at least log a tracking bug so that we go and add the necessary hooks in dotnet/arcade repo so that we can later ingest those changes here, and remove this file entirely so that we instead leverage the files from eng\common?

Reply:
Opened https://github.com/dotnet/arcade/issues/14470 .